libs/feature/search/src/lib/filter-dropdown/filter-dropdown.component.ts

```diff
@@ -46,10 +35,24 @@ export class FilterDropdownComponent implements OnInit {
 
   constructor(
     private searchFacade: SearchFacade,
-    private searchService: SearchService
+    private searchService: SearchService,
+    private aggregationsService: AggregationsService
   ) {}
 
   ngOnInit() {
+    this.choices$ = this.aggregationsService
+      .getAggregation(this.fieldName)
+      .pipe(
```

Comment:
I wonder if the pipe content shouldn't be part of the service method ?

Reply:
I would tend to not put it in the same method, as it could be reused (by the `OrganisationsService` for example). I added a commit for that to prevent code duplication. Then again, I'm not sure if architecturally the `feature/search` dependency in `feature/catalog` is disturbing, but I don't think so.
For the pipe content I could create a separate method in the service, but I'm not convinced it's really the better place as we are building the label of the component here.